$OpenBSD$

try fixing type swapping errors based on htobe64(3)
discussed with bcallah@: these inline functions may not be needed if endian.h
is included

Index: rpcs3/Crypto/utils.h
--- rpcs3/Crypto/utils.h.orig
+++ rpcs3/Crypto/utils.h
@@ -14,34 +14,7 @@
 #include "lz.h"
 #include "ec.h"
 
-// Auxiliary functions (endian swap, xor, prng and file name).
-inline u16 swap16(u16 i)
-{
-#if defined(__GNUG__)
-	return __builtin_bswap16(i);
-#else
-	return _byteswap_ushort(i);
-#endif
-}
-
-inline u32 swap32(u32 i)
-{
-#if defined(__GNUG__)
-	return __builtin_bswap32(i);
-#else
-	return _byteswap_ulong(i);
-#endif
-}
-
-inline u64 swap64(u64 i)
-{
-#if defined(__GNUG__)
-	return __builtin_bswap64(i);
-#else
-	return _byteswap_uint64(i);
-#endif
-}
-
+
 void xor_key(unsigned char *dest, const u8* src1, const u8* src2);
 inline void xor_key_sse(u8* dest, const u8* src1, const u8* src2)
 {
